### PR TITLE
Forward line information from trufflehog output

### PR DESCRIPTION
**Pull request by det**

### CODE REVIEW

**Comment by det** [comment]
@TylerJang27  Is out output somehow non-deterministic? Looks like it re-ordered some entries? Also is the `// trunk-upgrade-validation:RELEASE` line important?

**Review by puzzler7 — APPROVED** [review]
LGTM pending Tyler's approval

**Review by TylerJang27 — COMMENTED** [review]
> @ Tyler Is out output somehow non-deterministic? Looks like it re-ordered some entries? Also is the // trunk-upgrade-validation:RELEASE line important?

LGTM. Please remove the `// trunk-upgrade-validation:RELEASE` from the snapshot. That's only supposed to be there if the linter definition is stable but the snapshot was updated since our last release. By removing it, LUV will use the snapshot tagged at v0.0.21 rather than manually pulling this version in, allowing LUV to continue working as intended until the next release.

**Pull request merged**
